code/modules/ai/ai_holder_movement.dm

```diff
@@ -142,7 +142,7 @@
 	if(isturf(holder.loc) && can_act())
 		wander_delay--
 		if(wander_delay <= 0)
-			if(!wander_when_pulled && holder.pulledby)
+			if(!wander_when_pulled && (holder.pulledby | holder.grabbed_by.len))
```

Comment:
Out of curiosity, why `|` instead of `||` here?

Reply:
Heck just mixing up my ors again

Reply:
All good, something's gotta slip through once in a while.